app/views/report_mailer/tables_report.html.erb

```diff
@@ -5,7 +5,13 @@
 <%= @message %><br>
 <% @reports.each do |report| %>
   <% header, *rows = report.table %>
-  <h2><%= report.title %></h2>
+  <%# FIXME: Where is "Table 8" coming when no title is present? %>
+  <% if report.title && !report.title.match(/Table (\d)+/) %>
```

Comment:
This became too much of a distraction to delve into tonight, but if we leave `title:` blank on a section, it defaults to "Table 8" or whatever numbered table it is. I didn't find this code in a quick search, so for the moment we do this horrible thing.

Reply:
https://github.com/18F/identity-idp/blob/862cc1c5482ffe0b3bc2227a1bbce78b83173601/app/mailers/report_mailer.rb#L45-L47